source

```diff
@@ -99927,6 +99927,13 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
      character reference end state</span>.</p>
 
     <tr>
+     <td><dfn data-x="parse-error-disallowed-content-in-noscript">disallowed-content-in-noscript</dfn>
+     <td><p>This error occurs if the parser encounters content in a <code>noscript</code> element
+     that is not allowed by it's content model. In such a case, the parser implicitly closes the
```

Comment:
I think this is too overreaching to be accurate. For noscript in body in particular, you can put I think anything at all in noscript which violates the content model but does not close the noscript element. Maybe this needs to be limited to noscript in head? But even there you can have content model violations that doesn't close noscript (e.g., `<head><noscript><meta charset=utf-8>`). So maybe say something like "unexpected element in noscript in head".

Reply:
Yeah, I've changed it to:
```
This error occurs if the parser encounters any content other than ASCII whitespace, comments, a basefont element, a bgsound element, a link element, a meta element, a noframes element, or a style element within a noscript element. In this case, the parser implicitly closes the noscript element and processes the content as if it was placed after the noscript element in the markup.
```
in later commit. However, need to mention both in error code and text that it is related only to `noscript` in `head`.